pkgs/tools/package-management/cargo-edit/default.nix

```diff
@@ -1,27 +1,25 @@
-{ stdenv, pkgs, darwin, defaultCrateOverrides, fetchFromGitHub }:
+{ stdenv, lib, darwin, rustPlatform, fetchFromGitHub }:
 
-((import ./cargo-edit.nix { inherit pkgs; }).cargo_edit {}).override {
-  crateOverrides = defaultCrateOverrides // {
-    cargo-edit = attrs: rec {
-      name = "cargo-edit-${version}";
-      version = "0.2.0";
+rustPlatform.buildRustPackage rec {
+  name = "cargo-edit-${version}";
+  version = "0.3.0";
 
-      src = fetchFromGitHub {
-        owner = "killercup";
-        repo = "cargo-edit";
-        rev = "v${version}";
-        sha256 = "1jxppbb7s50pwg24qxf79fqvm1clwm2zdnv0xlkay7y05nd5bc0c";
-      };
+  src = fetchFromGitHub {
+    owner = "killercup";
+    repo = "cargo-edit";
+    rev = "v${version}";
+    sha256 = "0ngxyzqy5pfc0fqbvqw7kd40jhqzp67qvpzvh3yggk9yxa1jzsp0";
+  };
+
+  cargoSha256 = "1j7fqswdx6f2i5wr0pdavdvcv18j1l27a8ighr75p7f54apa27l8";
 
-      propagatedBuildInputs = stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+  propagatedBuildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```

Comment:
On linux `cargo-edit` also needs `pkgconfig` in `nativeBuildInputs` and `openssl` in `buildInputs`.

Reply:
Any reason why `darwin.apple_sdk.frameworks.Security` is not in `buildInputs`?

Reply:
`darwin.apple_sdk.frameworks.Security` is probably fine. `pkgs/build-support/rust/default-crate-overrides.nix` does the same.